frontend/talentsearch/src/js/components/Home/partials/Opportunities/Opportunities.tsx

```diff
@@ -146,19 +151,35 @@ const Opportunities = () => {
                 description: "Heading for executive jobs in government",
               })}
               link={{
-                href: "#", // TO DO: Get new path
-                label: intl.formatMessage({
-                  defaultMessage: "Browse exec jobs",
-                  id: "QcrDDA",
-                  description: "Link text to find executive jobs in government",
-                }),
+                href: profilePaths.myProfile(), // TO DO: Get new path
```

Comment:
Does the `// TO DO: Get new path` comment still apply or can it be removed?

Reply:
using `profilePaths.myProfile()` makes sense here, but can we change the value of `myProfile` in applicantProfileRoutes? Right now its `path.join("/", lang, "talent", "profile")`, I recommend `path.join("/", lang, "users", "me")`

Reply:
Actually, elsewhere on the page, the "create a profile" button either links directly to the user's profile or to `/login-info`, depending on whether the user is logged in or now. Should probably do the same here for consistency.

Reply:
> Should probably do the same here for consistency.

Shouldn't we change the other one to just link to profile? Since, if you are not logged in, it will just get you to login and send you back to the profile? If we link directly to login, and we change the default login redirect, the user will be taken there rather than to their profile. Also, it would be simpler than running conditional logic on all of our links.

Reply:
Removed comment and updated the paths in [clean up copy (whitespace, quotes)](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4442/commits/a739b51e2dc86a2cf4f4b20de407ec2995705665)

Reply:
Had to revert the change from `/talent/profile` -> `/users/me` since we have redirect logic in the app for this route. It would further complicate things and would require new tests to be created.

[revert profile path change](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4442/commits/9652cf60eeb243faea993e74713a235b682ece6c)

## Issue

Currently we are redirecting `/talent/profile` -> `/users/{userId}/profile`. In order to update the route, we would need to add another redirect for minimal improvements. introducing a new route that effectively does the same thing we would need both:

- `/talent/profile` -> `/users/{userId}/profile`
- `/users/me` -> `/users/{userId}/profile`

## Notes

While I am in agreement, we should remove the `/talent/profile` route, it seems out of scope of this issue. I think if we are going to look at updating routes, we should create a new ticket to look at them again and make sure they are what we want them to be. Once we launch, it is important we keep URLs as static as possible for a good UX and SEO.

Reply:
@tristan-orourke any thoughts on this?

Reply:
Yep, I agree. Seems like something we should do, also seems out of scope. @esizer could you define a new ticket for this?